ci/kokoro/windows/build-cmake-dependencies.ps1

```diff
@@ -20,39 +20,55 @@ $ErrorActionPreference = "Stop"
 if (-not (Test-Path env:CONFIG)) {
     throw "Aborting build because the CONFIG environment variable is not set."
 }
-$CONFIG = $env:CONFIG
+if (-not (Test-Path env:VCPKG_TRIPLET)) {
+    throw "Aborting build because the VCPKG_TRIPLET environment variable is not set."
+}
 
 $project_root = (Get-Item -Path ".\" -Verbose).FullName
+$vcpkg_dir = "cmake-out\vcpkg"
+$packages = @("zlib", "openssl",
+              "protobuf", "c-ares",
+              "grpc", "gtest", "crc32c", "curl",
+              "googleapis", "google-cloud-cpp-common[test]")
+$vcpkg_flags=@(
+    "--triplet", "${env:VCPKG_TRIPLET}",
+    "--overlay-ports=${project_root}/ci/kokoro/windows/vcpkg-ports")
+if ($args.count -ge 1) {
+    $vcpkg_dir, $packages = $args
+    $vcpkg_flags=("--triplet", "${env:VCPKG_TRIPLET}")
+}
 
 # Update or clone the 'vcpkg' package manager, this is a bit overly complicated,
 # but it works well on your workstation where you may want to run this script
-# multiple times while debugging vcpkg installs.  It also works on AppVeyor
+# multiple times while debugging vcpkg installs.  It also works on Kokoro
 # where we cache the vcpkg installation, but it might be empty on the first
 # build.
-Set-Location ..
 if (Test-Path env:RUNNING_CI) {
+    Remove-Item -LiteralPath "${vcpkg_dir}" -Force -Recurse
+}
+if (-not (Test-Path ${vcpkg_dir})) {
     Write-Host -ForegroundColor Yellow "`n$(Get-Date -Format o) Cloning vcpkg repository."
-    if (Test-Path "vcpkg") {
-        Remove-Item -LiteralPath "vcpkg" -Force -Recurse
-    }
-    git clone --quiet --depth 10 https://github.com/Microsoft/vcpkg.git
+    git clone --quiet --depth 10 https://github.com/Microsoft/vcpkg.git "${vcpkg_dir}"
```

Comment:
nit: The capital [M]icrosoft looks a little odd. I know it works, but I think the canonical URL for this repo is https://github.com/microsoft/vcpkg

Reply:
Done.